src/sage/schemes/curves/affine_curve.py

```diff
@@ -1828,20 +1889,30 @@ def braid_monodromy(self):
         each of this paths is the conjugated of a loop around one of the points
         in the discriminant of the projection of ``self``.
 
-        NOTE:
+        .. NOTE::
+
+            The projection over the `x` axis is used if there are no vertical asymptotes.
+            Otherwise, a linear change of variables is done to fall into the previous case.
+            This functionality requires the ``sirocco`` package to be installed.
```

Comment:
The sirocco note is not related to the first one. So if kept, it should be split.